### ISSUE TITLE
Filter buttons in igGrid control don't have proper aria-role specified

### ISSUE THREAD

**Issue by georgianastasov** [IgniteUI/ignite-ui#2240]
## Description  
The filter buttons in igGrid control don't have proper aria-role specified. This creates trouble when one is using accessibility tools such as NVDA. It reads the role wrongly in NVDA speech viewers. When reaching filter buttons, the screen reader cannot identify that it is a button or it functions as a button.

 * ignite-ui version: any
 * browser: any

## Steps to reproduce  

1. Go to this [sample](https://jsfiddle.net/georgianastasov/kn7ru52s/2/).
2. Use NVDA or other accessibility tool.
3. Reach the filter buttons in the grid.

## Result  
The screen reader doesn't identify the filter buttons as a buttons.

## Expected result  
The screen reader should identify the filter buttons as a buttons.

## Attachments  
![1](https://github.com/IgniteUI/ignite-ui/assets/48180072/3e6a4739-e984-40d0-b611-7f86ae0d14d8)

**Comment by saychakraborty89** [comment]
Thank you so much. Looking forward to that.
